.eslintrc.js

```diff
@@ -20,30 +20,30 @@ module.exports = {
     ],
     rules: {
         // These rules are only preliminary to ease the migration from tslint, many of them might make sense to have enabled!
-        "@typescript-eslint/camelcase": "off",
+        // "@typescript-eslint/camelcase": "warn",
```

Comment:
Is this deprecated now? https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/camelcase.md

Should we outright remove it?

Reply:
Let's remove it. We can consider doing another round of adding style rules later.